tests/integration-tests-topologies/src/main/java/org/apache/pulsar/tests/topologies/PulsarCluster.java

```diff
@@ -84,13 +98,17 @@ private PulsarCluster(PulsarClusterSpec spec) {
             .withEnv("pulsarNode", "pulsar-broker-0");
 
         this.csContainer = new CSContainer(clusterName)
+            .withLogConsumer(new Slf4jLogConsumer(log).withPrefix(CSContainer.NAME))
```

Comment:
you seem to enable log consumer at every container. this should be done at the common container class, rather than spreading it everywhere.

Reply:
will consolidate it

Reply:
this is not needed anymore, if log consumer is enabled in `PulsarContainer`.